pilot/pkg/proxy/envoy/v2/ads.go

```diff
@@ -507,6 +520,13 @@ func (s *DiscoveryServer) IncrementalAggregatedResources(stream ads.AggregatedDi
 }
 
 func (s *DiscoveryServer) pushAll(con *XdsConnection, pushEv *XdsEvent) error {
+	// Prevent 2 overlapping pushes. Disabled if push suppression is disabled
+	// (as fail-safe in case of bugs)
+	if suppressPush {
```

Comment:
This doesn’t seem to match with the comment above where suppresspush was meant to disable the behavior. Did you mean !suppressPush? (Also for more clarity, might be better to call this as allowConcurrentPushes)

Reply:
done